### ISSUE TITLE
undefined  function pcntl_signal()

### ISSUE THREAD

**Issue by maierwalter** [carlosvini/serve2#1]
Fatal error: Uncaught Error: Call to undefined function pcntl_signal()

I am trying to use it on Windows machine with php7,4. I tried to find the extension "pcntl", but couldn't find how to install it/enable it.

If someone knows a way, please let me know, it would reduce the time of loading a screen in my case by quite a lot.

Thanks for the package, I hope I will also be able to use it, would be very helpful.

**Comment by carlosvini** [comment]
I need to rewrite some parts for it to work on Windows, I'll try to do it when I have time. Are you using it with Laravel?

**Comment by maierwalter** [comment]
Yes, using it with Laravel, let me know if I can help in some way.

**Comment by carlosvini** [comment]
Wouldn't your problem be solved by installing Xampp (https://www.apachefriends.org/pt_br/index.html)?
